net/acme/files/run.sh

```diff
@@ -210,11 +210,14 @@ issue_cert()
 	local user_cleanup
 	local ret
 	local domain_dir
+	local use_custom_acme_directory
+	local custom_acme_directory_url
```

Comment:
Why two variables? Also, please use shorter variable names...

Reply:
Thanks, I will update the PR as requested. 

I have changes for this feature ready to push for the corresponding LuCI application, and added `use_custom_acme_directory` for a checkbox that conditionally shows/hides the input field for the URL (I don't want to show the input field, when the user doesn't need/understand this option).